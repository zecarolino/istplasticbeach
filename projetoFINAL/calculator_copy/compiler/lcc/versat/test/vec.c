void putchar(int ch) { asm("\trdw R11\n\taddi 2\n\twrw RB\n\trdwb\n\trdwb\n\twrw CPRT_BASE\n"); }
int vec[] = { 9, 7, 5, 3, 1, 0 };
int main() {
	int i;
	for (i = 0; vec[i]; i++)
		putchar(vec[i] + '0');
	putchar('\n');
	return 0;
}
