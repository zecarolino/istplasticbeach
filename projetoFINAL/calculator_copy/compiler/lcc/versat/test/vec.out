VCD info: dumpfile xtop.vcd opened for output.
97531
0
Catched Trap at time  940 clock cycles:
